tests/test_docker_compose_base.py

```
@@ -1,6 +1,4 @@
-"""
-Base test functionality for Docker Compose Qdrant tests
-"""
+"""Base test functionality for Docker Compose Qdrant tests."""
```

Ruff (0.12.2)

File `tests/test_docker_compose_base.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)